### DISCUSSION TITLE
What would a "data use standard" look like?

### DISCUSSION THREAD

**Discussion by stevieflow** [IATI/standard-process#11] [Ideas]
In our [last meeting](https://github.com/IATI/standard-process/blob/main/meetings/2024-10-22.md), we discussed that the existing IATI standard has been very focused on those that publish the data, rather than for people that use it.

There are some common and clear aspects of using IATI data that could benefit from agreed guideline and "standards".

Right now, there is little in place around this, and it is often left to those building tools and services to understand and undertake such processes by themselves, which seems to be a wasteful situation (after so many years)

Examples can include:

- How to split up financial data when an activity has multiple recipient-country/region values (see example below)
- How to convert currencies
- How to group together activities from the same publisher that are related (parent/child)
- How to group together activities from different publishers (using traceability functions)

**Questions to SSWG members:**

- Are there practises that would benefit from being standardised, or is this about better and clearer guidance?
- If this were a standard, would it need to be coupled with the existing artefacts, and be version controlled?
- Where and how do these standardised practices affect tools and services (regardless of who provides them) and would _they_ then be validated?

This is a side topic from our core discussions, but we know plenty of people have had experience and know-how around using IATI data.

**Comment by stevieflow** [comment]
Example: splitting transactions by multiple recipient countries/regions
--------------

Within the standard there are clear guidelines for using [multiple ``recipient-country/region`` values](https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/recipient-country/):

> Multiple countries and regions can be reported, in which case the percentage attribute MUST be used to specify the share of total commitments across all reported countries and regions.

This is something that has been followed by hundreds of organisations that publish IATI data, and is a key [ruleset in the validation](https://github.com/IATI/IATI-Rulesets/blob/version-2.03/rulesets/standard.json#L387).  

Indeed, this rule then points to further guidance, which is very [much aligned to publishing this data](https://iatistandard.org/en/guidance/standard-guidance/countries-regions/), aside from one glance towards use cases:

> Having one country per activity provides more disaggregated information, which is helpful for data users.

When it does come to *using* IATI activities that have multiple ``recipient-country/region``, and making accurate representation of the financial data there within, a person must take these percentages and use them "for splits" across the subsequent ``budget``, ``planned-disbursement`` and ``transaction`` elements.   

Currently, the IATI standard provides [no guidance on doing this](https://iatistandard.org/en/using-data/how-to-use-iati-data/), and so it is left to those that have attempted that to provide some documentation.  For example:

- Country Finance Development Data tool (CDFD) has an [outline of how this is achieved](https://countrydata.iatistandard.org/methodology/#24-splitting-transactions-for-multiple-sectors-and-countries) (thanks @markbrough)
- The COVID-19 tracker has a [similar methodology](https://data.humdata.org/viz-iati-c19-dashboard/about) (thanks @davidmegginson)

**Reply by HermanvanLoon** [reply]
Isn't the key question here if there is one dominant way to apply the country percentages to the (some of the) financial amounts? When there are multiple functional ways to do this we are not talking about a data use standard anymore. The application of different methodologies is in that case use-case specific (e.g. there are many ways to calculate a foreign currency amount since there are many exchange rate types).

**Reply by stevieflow** [reply]
Thanks @HermanvanLoon 

Yes - I think us "standardising" such practices has a clearer chance when they are unambiguous, as you suggest

I think the underlying issue is that we have historically left such process to users to figure out and do.  Hence, if we agree that something like the splitting of financial data has a clear and central methodology, then a) we should clearly document this for people and b) discuss whether there is a benefit to undertaking the service within the IATI infrastructure (making the data *easier* to interact with).  @robredpath will have thoughts

And agree - currency conversions can have different methods.  This *is* also something that is undertaken by various tools and services, but your point is well made.

Thanks again @HermanvanLoon !
